pkg/cmd/datamgr/cli/server/server.go

```diff
@@ -232,7 +234,7 @@ func newServer(f client.Factory, config serverConfig, logger *logrus.Logger) (*s
 		return nil, err
 	}
 
-	ctx, cancelFunc := context.WithCancel(context.Background())
+	//ctx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
Do we need to keep this commented out line here?

Reply:
Oops, thought I cleaned it up